subprojects/distributions/distributions.gradle

```diff
@@ -175,6 +188,17 @@ def srcZip = tasks.register("srcZip", Zip) {
     }
 }
 
+def docsZip = tasks.register("docsZip", Zip) {
+    archiveClassifier.set('docs')
+    into(zipRootFolder) {
+        with docsDistImage
+    }
+}
+
+tasks.register("buildDists") {
```

Comment:
Required as previously we were relying on `:distributions:integTest` to pull the dist task inside the task graph.